bloomfilter/_bloomfilter.pyx

```diff
@@ -146,9 +146,9 @@ cdef class RotatingBloomFilter:
     def rotate(self):
         '''Add a new bloom filter to our deque and remove any old bloom filters.'''
         self.bloom = BloomFilter(self.capacity, self.error_rate)
-        self.blooms.append(self.bloom)
+        self.blooms.appendleft(self.bloom)
```

Comment:
Considering that `blooms` is a public property via lines 142-144, this is an interface change. I'd recommend bumping the version number. In fact, the same could be said for #13, so you may want to bump the version twice. It doesn't look like this package is yet published to pypi, and I'm not sure how we're currently pinning versions internally, but that's something that should probably be addressed.